app/Repositories/Project/ProjectRepository.php

```diff
@@ -367,4 +372,258 @@ public function favoriteUpdate($authenticated_user, $project, $organization_id)
+     * Indexing of the projects
+     */
+    public function indexProjects($projects): void
+    {
+        if (empty($projects)) {
+            return;
+        }
+        // first get the collection of playlists - needed in activities update
+        $playlists = Playlist::whereIn('project_id', $projects)->get('id');
+
+        // search-able is needed as on collections update observer will not get fired
+        // so searchable will updated the elastic search index via scout package
+        // update directly on query builder
+        $this->model->whereIn('id', $projects)->searchable();
+
+        // to fire observer update should be done on each single instance of models
+        // $projects->each->update(); can do for firing observers on each model object - might need for elastic search
+        // prepare the query builder from collections and perform update
+        Playlist::whereIn('project_id', $projects)->searchable();
+
+        // update related activities by getting keys of parent playlists
+        Activity::whereIn('playlist_id', $playlists->modelKeys())->searchable();
+    }
+
+    /**
+     * @param $projects
+     * @param $flag
+     * @return string
+     * @throws GeneralException
+     */
+    public function toggleStarter($projects, $flag): string
+    {
+        if (empty($projects)) {
+            throw new GeneralException('Choose at-least one project.');
+        }
+        $this->model->whereIn('id', $projects)->update(['starter_project' => (bool)$flag]);
+        return 'Starter Projects status updated successfully!';
+    }
+
+    /**
+     * To export project and associated playlists
+     *
+     * @param $authUser
+     * @param Project $project
+     * @param int $suborganization_id
+     * @throws GeneralException
+     */
+    public function exportProject($authUser, Project $project)
+    {
+        $zip = new ZipArchive;
+        
+        $project_dir_name = 'projects-'.uniqid();
+        Storage::disk('public')->put('/exports/'.$project_dir_name.'/project.json', $project);
+        
+        $project_thumbanil = "";
+        if (filter_var($project->thumb_url, FILTER_VALIDATE_URL) == false) {
+            $project_thumbanil =  storage_path("app/public/" . (str_replace('/storage/', '', $project->thumb_url)));
+            $ext = pathinfo(basename($project_thumbanil), PATHINFO_EXTENSION); 
+            if(file_exists($project_thumbanil)) {
+                Storage::disk('public')->put('/exports/'.$project_dir_name.'/'.basename($project_thumbanil),file_get_contents($project_thumbanil));
+            }
+        }
+       
+        $playlists = $project->playlists;
+        
+        foreach ($playlists as $playlist) {
+            
+            $title = $playlist->title;
+            Storage::disk('public')->put('/exports/'.$project_dir_name.'/playlists/'.$title.'/'.$title.'.json', $playlist);
+            $activites = $playlist->activities;
+            ;
+            foreach($activites as $activity) {
```

Comment:
; -- should be removed line 506
And formatting issue. Need spaces after concatenation.